src/main/groovy/edu/oregonstate/mist/personsapi/mapper/PersonMapper.groovy

```diff
@@ -14,6 +14,17 @@ import java.sql.SQLException
 public class PersonMapper implements ResultSetMapper<PersonObject> {
     PhoneFormatter phoneFormatter = new PhoneFormatter()
     public PersonObject map(int i, ResultSet rs, StatementContext sc) throws SQLException {
+        String rawSSN = rs.getString('SSN')
+        String ssnStatus = 'invalid'
+
+        if (rawSSN && rawSSN.startsWith('VAULT')) {
+            ssnStatus = 'valut'
+        } else if (rawSSN == 'REDACTED' || rawSSN == 'ARCHIVED') {
```

Comment:
In his email, Reed said "is “REDACTED”, “ARCHIVED”, or starts with “VAULT”, then the SSN is in the vault" so I think this should actually be vault also.

Reply:
Right, I originally set them all as `vault` too, but thought about it more, since `REDACTED` and `ARCHIVED` were more specific and clearer statuses than just 'in vault', I eventually decided to have different statuses for them. @fernmig what do you think?

Reply:
Reed mentioned `REDACTED` are `ARCHIVED` are all in the vault now, based on our discussion, I'm going to hide these detail codes for now unless there is another need from our client in the future.

Reply:
Sounds good. With the changes to the query means this code for ssnStatus can go away and all you need is "ssnStatus = rawSSN.toLowerCase()".